backend/src/test/java/de/jmpsoftware/backend/model/PipeEntryTest.java

```diff
@@ -0,0 +1,112 @@
+package de.jmpsoftware.backend.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PipeEntryTest {
+
+    @Test
+    void PackedInteger()
```

Comment:
method names should start with lowercase, maybe add a bit more description what the test does. @DisplayName is your friend (I mean the annotation, though there probably is a github user with that name)